The Fish game.md

```diff
@@ -0,0 +1,55 @@
+# Summary
+This task works to use collective sensing in groups of fish in order to better understand collective intelligence in human groups. To do this, individuals participated in a gradient-sensing task in which every location in this virtual world responded to a score value that changed over time.
+
+# References
+Main paper: https://people.csail.mit.edu/pkrafft/papers/krafft-et-al-2015-emergent.pdf
+
+Supporting paper: https://www-jstor-org.proxy.library.upenn.edu/stable/pdf/23343730.pdf?refreqid=excelsior%3A497fb73bd3cb7e6d7fbdb013b9e0a0f4
+
+# Stimuli
+## The visual components
+![Fish1](images/Fish1.png)
+![Fish2](images/Fish2.png)
+
+> Participants could only see the scores at their avatars' locations
+> 
+> They could also see the positions, directions, and speeds of all other players
+> 
+> Players controlled their avatars using the left and right arrow keys to turn and could hold the space bar to accelerate
+
+## Materials for alternative versions of the experiment 
+The weighting between the noise field and the spotlight were manipulated to generate different task conditions
+
+# Procedure
+## Steps
+> Participants either played alone or in groups from 2-6.
+> 
+> After agreeing to participate, participants were presented with a set of instructions. 
+> > These instructions described the mechanics of the game
+> > 
+> The participants were not informed about the nature of the underlying score fields and were not encouraged to work together
+> 
+> Participants completed a comprehension test
+> 
+> Participants were put into a waiting room for up to five minutes or until a pre-assigned number of other players joined the game
+> 
```

Comment:
While in the waiting room, participants could familiarize themselves with the controls of the game. Players were not shown any score in the waiting room unless the participant was against a wall, in which case the displayed score would change from a dashed line to a red “0%”.